ArcGIS.md

```diff
@@ -1,4 +1,28 @@
-# ArgGIS #
+# ArgGIS Tutorials #
+
+## What is GIS? ##
+A geographic information system is used to display various types of data within a map-like format. GIS can connect various types of local data that display the physical geographical locations, as well as additional descriptions. 
+## ArcGIS Online ##
+Is a cloud-based software that allows you to build interactive visual maps. This is Esri’s web-based mapping platform. You will need a License through ArcGIS in order to use it and thankful we have access to ArchGIS through Penn State!
+
+## ArcGIS Pro ##
+This is Esri’s desktop application for ArcGIS. Please keep in mind the ArcGIS was made for Windows, not Macs. 
+
+## How to Create an Account and Download ArcGIS Pro: ##
+1. Go to [ArcGIS Online](https://www.esri.com/en-us/arcgis/products/arcgis-online/overview) and click the Sign In button.
+2. You will want to sign with the ArcGIS Organization’s URL section.
+    1. You will want to type in the word pennstate
+    2. Now this will take you to the Penn State login page. This page will prompt you to login in with your PSU account.
+3. Once you are logged into your ArcGIS account that is registered through Penn State go to the top right hand corner of your screen and click your account tab. Once the drop menu is open, click the My settings section.
+4. Then you will want to click the Licenses tab near the top left corner of your webpage.
+5. Here you will see all Licenses Penn State has to offer for any PSU students that want to use ArcGIS.
+6. Scroll down and find the ArcGIS Pro License click the download link. Once the download has completed, click and open the downloaded ArcGIS file.
+    1. You will have a pop-up window that will guide you through a series of steps for the installation process.
+    2. Choose the desired destination where you would like to keep your ArcGIS desktop program. Then press the Next button.
+7. After the extraction process is completed make sure the *Launch the setup program* box is checked. :heavy_check_mark: 
+8. Then press the Close button.
+## Formatting Data for ArcGIS: ##
+## How to Look up Data within Living Atlas: ##
+## Adding Data to your Map: ##
+## Display Data and Tables on your Map: ##
```

Comment:
@Tiny-Pickles These sections look just right to me so far! We may want to add more as you dig into analyzing correlations in your data sets, as in, being able to highlight which reservations are close to corporate camps, etc?